### ISSUE TITLE
can't determine image format

### ISSUE THREAD

**Issue by harrisonmg** [harrisonmg/jigsaw#8]
```
Is "https://images3.alphacoders.com/708/708114.jpg" the correct image URL?

Underlying error: The image format could not be determined', crates/server/src/main.rs:51:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread 'main' panicked at 'Error loading puzzle

Is "https://images.alphacoders.com/309/309813.jpg" the correct image URL?

Underlying error: The image format could not be determined', crates/server/src/main.rs:51:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread 'main' panicked at 'Error loading puzzle

Is "https://images4.alphacoders.com/132/1324789.png" the correct image URL?

Underlying error: The image format could not be determined', crates/server/src/main.rs:51:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread 'main' panicked at 'Error loading puzzle

Is "https://images4.alphacoders.com/125/1259382.jpg" the correct image URL?

Underlying error: The image format could not be determined', crates/server/src/main.rs:51:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread 'main' panicked at 'Error loading puzzle

Is "https://images.alphacoders.com/206/206551.jpg" the correct image URL?

Underlying error: The image format could not be determined', crates/server/src/main.rs:51:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by harrisonmg** [comment]
this is actually a mistaken error message on my part

looks like I'm running into cloudflare's hotlink protection or something and getting a code 403

It looks like it might be time to load images manually

**Comment by harrisonmg** [comment]
should be fixed with local image loading by 805cbdb
